_plugins/flag_tag.rb

```diff
@@ -0,0 +1,16 @@
+require "liquid"
+
+module Jekyll
+  class FlagTag < Liquid::Tag
+    def initialize(tag_name, country_code, tokens)
+      super
+      @country_code = country_code
+    end
+
+    def render(context)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused method argument - `context`. If it's necessary, use `_` or `_context` as an argument name to indicate that it won't be used. You can also write as `render(*)` if you want the method to accept any arguments but don't care about them.](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5610789)